test/e2e/networks/ci.toml

```diff
@@ -5,8 +5,7 @@ ipv6 = true
 initial_height = 1000
 vote_extensions_update_height = 1004
 vote_extensions_enable_height = 1007
-pbts_update_height = 1006
-pbts_enable_height = 1009
+pbts_enable_height = 1000
```

Comment:
```suggestion
pbts_update_height = -1
pbts_enable_height = 1000
```

Reply:
This is important.
Default value of `pbts_update_height` is absent is 0, which is `InitChain`, but we can to set it on **genesis** (-1)